README.md

```diff
@@ -16,29 +20,24 @@ It deploys:
   - An access policy to allow the MI to read secrets
   - an SSH key and uploads
 
+- An `aci-devops-agents` module that creates:
+  - data structures for the vnet/subnet created before
+  - an `azurerm_network_profile` and `azurerm_container_group` that deploy a container running the Azure DevOps agents in the vnet (which requires delegation to `Microsoft.ContainerInstance/containerGroups`)
+  - The agents autamatically register themsleves with Azure DevOps.
+
+  The pipeline in `src/pipelines/infra.yaml` applies the content on the `main.tf` in `src/terrafomr` which calls the `common` module and the `aci-devops-agents` module in sequence; after that, a call to the terrafom template in `src/terraform/delete_access_policy` eliminates the access policy for the `current` service connection (SC) service principal (SP) effectively removing access to the secret to subsequent Azure DevOps runs. This way, even by accessing the SC' SP will make it impossible to access the secrets in Keyvault.
+
+
+### Extra:
+
+We added (under `src/scripts`):
 
+- A script to populate a Variable group from a set of key/values (TODO: use a YAML formatted file as input): `create_vg.sh`
+- A python module to delete agents from a pool. As the [official Azure DevOps CLI extension for Azure CLI](https://github.com/Azure/azure-devops-cli-extension) does not suppoer yet ([issue](https://github.com/Azure/azure-devops-cli-extension/issues/955)) we thought it would be handy to have a module to implemnt such functionality in a script (TODO: implement the module in a cli command)
```

Comment:
Spelling error in the second sentence, I'm guessing it was meant to be "does not support it yet"

Reply:
@ams0 this is for you.

Reply:
addressed in #13